.github/workflows/ci.yml

```diff
@@ -75,7 +72,7 @@ jobs:
       - name: Setup dart
         uses: dart-lang/setup-dart@v1
       - name: publish dry run
-        run: cd packages/dart && dart pub publish --dry-run
+        run: dart pub publish --dry-run
   check-flutter:
```

Comment:
Delete `check-flutter` job

Reply:
This job was deleted but this section had many codes, are you sure?